charts/kubescape-operator/templates/storage/deployment.yaml

```diff
@@ -48,6 +48,8 @@ spec:
           - name: "GOMEMLIMIT"
             value: "{{ .Values.storage.resources.requests.memory }}B"
           {{- if $components.otelCollector.enabled }}
+          - name: ACCOUNT_ID
+            value: "{{ .Values.account }}"
```

Comment:
It is best practice to use `valueFrom` so you can load the value from the secret